src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -341,16 +348,22 @@ public static class SDKRestClient implements Closeable {
 
         private final SDKClient sdkClient;
         private final RestHighLevelClient restHighLevelClient;
+        private final String token;
+        private final String headerName;
 
         /**
          * Instantiate this class wrapping a {@link RestHighLevelClient}.
          *
          * @param sdkClient The SDKClient instance.
          * @param restHighLevelClient The client to wrap.
+         * @param token The bearer token.
+         * @param headerName The header name.
          */
-        public SDKRestClient(SDKClient sdkClient, RestHighLevelClient restHighLevelClient) {
+        public SDKRestClient(SDKClient sdkClient, RestHighLevelClient restHighLevelClient, String token, String headerName) {
```

Comment:
just leave this as is

Reply:
@dbwiddis I don't understand how the sdkclient communicate with opensearch, could you please give some explanations about this?